unlock-app/src/components/lock/Lock.js

```diff
@@ -82,6 +86,7 @@ Lock.defaultProps = {
 
 const mapDispatchToProps = dispatch => ({
   purchaseKey: key => dispatch(purchaseKey(key)),
+  updateLockPrice: (lock, price) => dispatch(updateLockPrice(lock, price)),
```

Comment:
I have no idea how this should actually be wired up. I just saw something close to what looked correct and put a placeholder in here for now.

Reply:
This is wired up ;) Basically the function `mapDispatchToProps ` is courtesy of `react-redux` and is a way to invoked reducer actions on the front end application.